### ISSUE TITLE
Steel Sheet

### ISSUE THREAD

**Issue by DragonGirl7567** [Lucreeper74/Create-Metallurgy#22]
**Is your feature request related to a problem? Please describe.**
No problem, just a suggestion for mods that require a steel sheet in recipes

**Describe the solution you'd like**
Suggest making a sheet version of the steel

**Describe alternatives you've considered**
Create: Alloyed does have steel sheets, however, it is not updated to 1.20.1

Sorry to bother

**Comment by Lucreeper74** [comment]
Hey @DragonGirl7567 :wave:
This could be a relevant content addition as steel is used in many other mods. This will be implemented, I don't know when yet. I am currently working on a mineral upgrade. I'll add it along the way. I'll keep you posted, and I'll leave this post open until I push the change.

And, no worries you don't bother me!

**Comment by AraHaan** [comment]
I wish this sheet can be usable in places of tungsten ones as on our server it is so ultra rare that it is impossible to get even in all the mining dimensions. And sadly cannot reset them as a few people have built their bases in the mining dimension/ set up their infinite supply of all of the metals except for tungsten (we have it to where the hose pulley considers bodies of liquids over 10,000 source blocks to be considered a bottomless supply. Likewise if it was possible to create tungsten in game by having an alloy machine recipe with like 1 gold and 2 steel ingots for 3 tungsten ingots it could be an option for our server as well (please so it cannot be impossible to get anymore on our server).

**Comment by Lucreeper74** [comment]
It's weird, it's not supposed to be that rare normally, just found below y=65 in the Nether. But However, I had not considered the fact that the pump could make liquids infinite. I will correct this, but don't worry, I will leave the choice with config to be able to allow it. I will experiment with Tungsten ore on local server to see. And also, i wont make an alloy recipe for the Tungsten cause its not an alloy.

**Comment by AraHaan** [comment]
> It's weird, it's not supposed to be that rare normally, just found below y=65 in the Nether. But However, I had not considered the fact that the pump could make liquids infinite. I will correct this, but don't worry, I will leave the choice with config to be able to allow it. I will experiment with Tungsten ore on local server to see. And also, i wont make an alloy recipe for the Tungsten cause its not an alloy.

The infinite from hose pulley's is a setting in base create that is by default not enabled. Once it is enabled it can make liquids over 10,000 source blocks to be considered bottomless.

**Comment by Lucreeper74** [comment]
Yes, I will made an option to only enable it for molten metals. To let player choose between lava/water and molten metals or just lava/water
